usecases.md

```diff
@@ -140,6 +140,89 @@ The table below shows the attributes for three different examples. We assume all
 | Site | Sensor X (America/Denver) | Power Plant Y (Etc/GMT+7) | Aggregate Z (America/Phoenix) |
 
 
+### Probabilistic forecasts {#probforecastdef}
+{: .anchor}
+
+The Solar Forecast Arbiter uses cumulative distribution functions (CDF) to
+quantify probabilistic forecasts. A CDF is represented here as a sequence of
+(*variable value*, *percentile*) pairs, where percentiles range from 0 to 100.
+
+Each [forecast data point](#forecastdatapoint) describes one (variable value, percentile)
```

Comment:
I'm OK with the description except perhaps this sentence, which we can discuss so I understand better what you intend.